src/App.js

```diff
@@ -1,20 +1,441 @@
 import React from "react";
-import logo from "./logo.png";
+
 import "./App.css";
+import Header from "./components/Header";
+
+import TaskList from "./components/TaskList";
+import TaskListInProgress from "./components/TaskListInProgress";
+import TaskListInReview from "./components/TaskListInReview";
+import TaskListCompleted from "./components/TaskListCompleted";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tasks: [
+        {
+          id: 0,
+          title: "Reverse Engineer the Type-33 Needler",
+          task: "Coding a software to reverse engineer the Needler for Sacred Promissory",
+        },
+        // {
+        //   id: 1,
+        //   title: "Fixing bugs",
+        //   task: "Fixing coding bugs on the software for Machine Learning AI",
+        // },
+      ],
+      tasksInProgress: [
+        {
+          // id: 0,
+          // title: "React Website - ORION Project",
+          // task: "Coding a frontend website for Doctor Halsey",
+        },
+      ],
+      tasksInReview: [
+        // {
+        //   id: 0,
+        //   title: "Project Fireteam Osiris",
+        //   task: "Implementing a MongoDB Database for Spartan Locke",
+        // },
+      ],
+      tasksCompleted: [
+        {
+          // id: 0,
+          // title: "Improved M-41 SPNKER with upgrades to tracking",
+          // task: "Added tracking to lock on to targets with greater distance and speed of the payload",
+          // timeTaken: "",
+          // keyTakeaways: "",
+        },
+        // {
+        //   id: 1,
+        //   title: "Fixed software",
+        //   task: "Fixing coding bugs on the software for Machine Learning AI",
+        // },
+      ],
+    };
+  }
+
+  // Transfer tasks from Open Tasks to Tasks In Progress
```

Comment:
This is a redundant comment. The function name already documents this, so we do not need additional sentences to describe what the function is doing